improve docs intro (#1037)
JuliaRobotics/Caesar.jl: docs/src/index.md

</p>
```

## Introduction

> Caesar.jl facilates software development for spatial/physical AI from multiple sensor data, and multiple sessions or human / semi-autonomous / autonomous agents.  This software is being developed with broadly [Industry 4.0](https://en.wikipedia.org/wiki/Fourth_Industrial_Revolution), AI, data analytics, robotics, and [Work of the Future](https://workofthefuture.mit.edu/) in mind.  Caesar.jl is an "umbrella package" to combine many other libraries from across the Julia package ecosystem.  

Author: ```suggestion
Caesar.jl facilates software development for spatial/physical AI from multiple sensor data, and multiple sessions or human / semi-autonomous / autonomous agents.  This software is being developed with broadly [Industry 4.0](https://en.wikipedia.org/wiki/Fourth_Industrial_Revolution), AI, data analytics, robotics, and [Work of the Future](https://workofthefuture.mit.edu/) in mind.  Caesar.jl is an "umbrella package" to combine many other libraries from across the Julia package ecosystem.  Additionally the NavAbilitySDKs allow various multi-language and cost-sharing features for large networks, compute loads, and expert help.
```

> Caesar.jl facilates software development for spatial/physical AI from multiple sensor data, and multiple sessions or human / semi-autonomous / autonomous agents.  This software is being developed with broadly [Industry 4.0](https://en.wikipedia.org/wiki/Fourth_Industrial_Revolution), AI, data analytics, robotics, and [Work of the Future](https://workofthefuture.mit.edu/) in mind.  Caesar.jl is an "umbrella package" to combine many other libraries from across the Julia package ecosystem.  Additionally the NavAbilitySDKs allow various multi-language and cost-sharing features for large networks, compute loads, and expert help.

Author: ```suggestion
Caesar.jl facilates software development for spatial/physical AI from multiple sensor data, and multiple sessions or human / semi-autonomous / autonomous agents.  This software is being developed with broadly [Industry 4.0](https://en.wikipedia.org/wiki/Fourth_Industrial_Revolution), AI, data analytics, robotics, and [Work of the Future](https://workofthefuture.mit.edu/) in mind.  Caesar.jl is an "umbrella package" to combine many other libraries from across the Julia package ecosystem.  Additionally, the [NavAbilitySDKs](https://github.com/NavAbility/) allow for multi-language and cost-sharing features.  Commercial support is available where large networks, multiple compute loads, and expert help can help drastically reduce cash-burn.
```

## TL;DR; Science Underpinning

> Caesar.jl is primarilly concerned with data processing / algorithmic / AI-related software for data-fusion, perception, contradiction extraction, navvigation affordances, surrogates, geometric representation, sensor calibration, solution verification, efficient solving, equipment interactions, swarms, and related topics.  At the highest level, the entire purpose of CJL and co are for the most engineering sensible and power solutions to the following expressions:

Author: ```suggestion
Caesar.jl is primarilly concerned with data processing / algorithmic / AI-related software for data-fusion, perception, contradiction extraction, navigation affordances, surrogates, geometric representation, sensor calibration, solution verification, efficient solving, equipment interactions, reality capture, digital twins, swarms, and related topics.  At the highest level, the entire purpose of CJL and co are for the most engineering sensible and power solutions to the following expressions:
```

> Caesar.jl is primarilly concerned with data processing / algorithmic / AI-related software for data-fusion, perception, contradiction extraction, navigation affordances, surrogates, geometric representation, sensor calibration, solution verification, efficient solving, equipment interactions, reality capture, digital twins, swarms, and related topics.  At the highest level, the entire purpose of CJL and co are for the most engineering sensible and power solutions to the following expressions:

Author: ```suggestion
Caesar.jl is primarilly concerned with data processing / algorithmic / AI-related software for data-fusion, perception, contradiction extraction, navigation affordances, surrogates, same-day-GPS-replacement, geometric representation, sensor calibration, solution verification, efficient solving, equipment interactions, reality capture, digital twins, swarms, and related topics.  At the highest level, the entire purpose of CJL and co are for the most engineering sensible and power solutions to the following expressions:
```

```
# the "measurement process/data collection manifestation/reality/collapse" is (i.e. robot drives around somewhere)
observation_states + model_uncertainty = convolve(physics_models, observation_uncertainty)

> # model_uncertainty = quantum_uncertainty

Author: ```suggestion
# model_uncertainty = quantum_uncertainty + measurement_error
```

> # while the "solve graph" (aka inversion) manifestation (aka data-fusion /state estimation) is:

Author: ```suggestion
# while the "solve graph" manifestation (aka inversion / data-fusion / state estimation / inference) is:
```

estimated_states + model_uncertainty = deconvolve(best_models, observation_states)

> # model_uncertainty = quantum + modeling_error + numerical + ...

Author: ```suggestion
# model_uncertainty = (quantum + modeling_error) + numerical + ...
```

```

## Open-core, Community and Commercial

> Click here to go to the Caesar.jl (permisively licensed) Github repo:

Author: ```suggestion
Click here to go to the Caesar.jl (permissively licensed) Github repo:
```

[![source](https://img.shields.io/badge/source-code-yellow.svg)](https://github.com/JuliaRobotics/Caesar.jl)

> [WhereWhen.ai's NavAbility products and services](https://www.wherewhen.ai) continues to develop the Caesar.jl suite of open-source libraries.  Please reach out for any additional information (info@navability.io), or using the community links provided below.

Author: ```suggestion
[WhereWhen.ai's NavAbility products and services](https://www.wherewhen.ai) continue to develop the Caesar.jl suite of open-source libraries.  Please reach out for any additional information (info@wherewhen.ai), or using the community links provided below.
```
